Exercise solution.ipynb

```diff
@@ -0,0 +1,250 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
```

Comment:
The format of this file indicates @frank-engel-usgs used jupyter notebook cells to propose his solution